src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/model/AgencyInternalLocation.java

```diff
@@ -95,6 +99,14 @@ private boolean hasSpace() {
         return capacity != null && currentOccupancy != null && currentOccupancy < capacity;
     }
 
+    public Integer incrementCurrentOccupancy() {
+        if (currentOccupancy != null) {
```

Comment:
Not sure how currentOccupancy could be null - but if so should we set it to 1 here?

Reply:
agreed